### ISSUE TITLE
formatToParts in browsers (partly) supporting Intl API

### ISSUE THREAD

**Issue by pkozlowski-opensource** [andyearnshaw/Intl.js#260]
Executing `new Intl.DateTimeFormat('en-US').formatToParts(new Date()));` in a browser that has native `Intl` API implementation (Chrome, FFox, ...) results in:

> TypeError: (intermediate value).formatToParts is not a function

Would it make sense for this polyffill to monkey-patch the Intl object so new methods like `formatToParts` can be used in browsers already having `Intl` API partly implemented? Or is this considered [gross](https://github.com/sindresorhus/ponyfill#how-are-ponyfills-better-than-polyfills)? 

Is using `IntlPolyfill` a preferred way to go then?

**Comment by pkozlowski-opensource** [comment]
I guess this is related to #242

**Comment by caridy** [comment]
yes, that's correct! any help on that direction will be appreciated :)

**Comment by icambron** [comment]
I'm having some trouble imagining how that would work. Would you have to, like, parse the result you got from the native Intl impl?
